src/ui/compilationStatusView.ts

```diff
@@ -27,10 +27,15 @@ function toStatusMessage(status: CompilationStatus, message?: string | null): st
   }
 }
 
+interface DocumentStatusMessage {
```

Comment:
```suggestion
interface IDocumentStatusMessage {
```
So far we've got the convention to prefix interfaces with `I`.